velox/functions/prestosql/Reduce.cpp

```diff
@@ -169,6 +170,11 @@ class ReduceFunction : public exec::VectorFunction {
           &localResult);
     }
     context.moveOrCopyResult(localResult, rows, result);
+    for (int i = 0; i < rows.end(); i++) {
+      if (isNullArrays[i]) {
+        result->setNull(i, true);
```

Comment:
```result->setNull(i, isNullArrays[i])```

Reply:
Maybe `setNull(i, true)` is more clear for code reading.